packages/lib-core/package.json

```diff
@@ -22,9 +22,11 @@
     "test": "echo 'Write some tests!'"
   },
   "peerDependencies": {
-    "lodash-es": "^4.17.21",
     "react": "^17.0.2",
-    "redux": "^4.1.2",
+    "redux": "^4.1.2"
+  },
+  "dependencies": {
+    "lodash-es": "^4.17.21",
```

Comment:
I'm in favour of this being a `dependency` -- the users of lib-core package shouldn't need to bring lodash into their dependencies.

Reply:
Spoke with @karelhala about this earlier, for utility-like dependencies which meet both of these conditions

- does not use module-level state (which would cause two different instances to use two different states)
- does not have to be exactly the same version vs. what the actual plugin or host application uses at runtime

it's fine to put them into `dependencies` and have them pulled into the consuming project's `node_modules` structure.

Reply:
> the users of lib-core package shouldn't need to bring lodash into their dependencies

Right. Also, it's completely fine if users would use a different version of Lodash if they wanted to.